infrastructure/lambda-search-doc-references.tf

```diff
@@ -0,0 +1,39 @@
+module "search-document-references-gateway" {
```

Comment:
the lambda and gateway needs to be referenced in 'aws_api_gateway_deployment' in triggers and depends_on